src/tools/compiletest/src/directives/cfg.rs

```diff
@@ -164,6 +164,12 @@ fn parse_cfg_name_directive<'a>(
         message: "when the architecture is part of the Thumb family"
     }
 
+    condition! {
+        name: "uefi",
+        condition: config.target.ends_with("uefi"),
+        message: "when target platform is UEFI"
+    }
```

Comment:
Question: is `uefi` a `target_env`? Can we use that instead of doing substring matching against the target tuple?

Reply:
It is `cfg!(target_os = "uefi")`, so actually the condition here is not needed. I thought it was after some errors earlier.